### PR TITLE
Parse JSON config from file instead of environment

### PR DESCRIPTION
**Pull request by GordonDiggs**
/cc @codeclimate/review

### CODE REVIEW

File: bin/bundler-audit [hunk 1]
```diff
@@ -4,10 +4,12 @@ $LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "../lib"))
 
 require 'cc/engine/bundler_audit'
 
-def parse_engine_config(raw)
-  raw ? JSON.parse(raw) : {}
+engine_config = {}
+
+if File.exists?("/config.json")
+  engine_config = JSON.parse(File.read("/config.json"))
 end
```
Comment (pbrisbin) [inline bin/bundler-audit]: I've seen this pattern in a few places, I think we should decide on a style:

```rb
x = default

if condition
  x = something
end
```

Vs

```rb
if condition
  x = something
else
  x = default
end
```

I personally prefer the second, but I'm OK if everyone else prefers what you're doing.

No need to discuss this now, just wanted to raise it when an example came by.

**Comment by pbrisbin** [comment]
LGTM

File: bin/bundler-audit (hunk 1 above)
Comment (GordonDiggs) [inline bin/bundler-audit]: I like the second option - will implement that. Ruby doesn't make a new scope for if/else, so the third is a weird pattern to me.

Comment (pbrisbin) [inline bin/bundler-audit]: :+1:

**Pull request merged**